Jenkinsfile

```diff
@@ -22,117 +22,126 @@ pipeline {
       }
     }
 
-    stage('Build client Docker image') {
-      steps {
-        sh './bin/build'
+    stage('Build and test Secrets Provider') {
+      when {
+        expression {
+         sh(returnStatus: true, script: 'git diff origin/master --name-only | grep -v "^.*\\.md$" > /dev/null') == 0
+        }
       }
-    }
-
-    stage('Scan Docker Image') {
-      parallel {
-        stage("Scan Docker Image for fixable issues") {
+      stages {
```

Comment:
why stages?

Reply:
I did similar changes to the `conjur` PR: https://github.com/cyberark/conjur/pull/1939
In short, it needs a `stages` group to add the `when` on all steps need skipping.
These are all testing, excluding changelog step that still need running.